e107_languages/English/admin/lan_theme.php

```diff
@@ -115,11 +115,12 @@
 //marketplace
 define("TPVLAN_81","Downloading..."); 
 define("TPVLAN_82","Download Complete!");
-define("TPVLAN_83","Automated download not possible.");
+define("TPVLAN_83","Automated download not possible! Please [x]");
 define("TPVLAN_84","Please Download Manually");
```

Comment:
It's close. I believe what is needed here is. 
define("TPVLAN_83","Automated download not possible!");
define("TPVLAN_84","[Please Download Manually]");

..then use search/replace on [ and ] to add the link (`<a href=... >`) and ( `</a>` )